db/migrate/20220721103300_add_role_to_users.rb

```diff
@@ -0,0 +1,5 @@
+class AddRoleToUsers < ActiveRecord::Migration[7.0]
+  def change
+    add_column :users, :role, :string, default: 'normal', null: false
+  end
+end
```

Comment:
- [ ] I received the attached error when I tried migrating your database with the command; `rails db:migrate` after creating the database. Kindly fix the error so we can test the `CanCanCan` functionality on your app.

<img width="314" alt="Code_gkCH0U1QfL" src="https://user-images.githubusercontent.com/81468675/180235448-c8427678-1dca-41ba-a77a-b081c75fb87e.png">

Reply:
Thank you for the feedback. We fixed the error.